wagtail_wordpress_import/importers/wordpress.py

```diff
@@ -304,6 +303,23 @@ def connect_page_categories(self, page, category_model, item):
             page.categories = page_categories
 
 
+def default_prefilters():
```

Comment:
1. **Comment**: As discussed elsewhere, I agree this should change from a function to a constant.
2. **Question**: I also commented about functions vs constants at https://github.com/torchbox/wagtail-wordpress-import/pull/106. I'm confused about which conversation was which. Was I wrong there?